cogs/assistance-cmds/combo.rosalina.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Rosalina combo
+help-desc: Key combo to open the Rosalina menu
+---
+Press the following keys at the same time to open Rosalina on modded consoles:
+
+<:3ds_button_l:295004499511214080> + <:3ds_dpad_down:295004561670930432> + SELECT
```

Comment:
This may not always be the case for people who need config.ini hacks for broken buttons

Reply:
> This may not always be the case for people who need config.ini hacks for broken buttons

Yeah true, mention it's the stock combination with a blurb about it might have been changed if buttons don't work?

Reply:
Probably the best way to go if we decide to add this